Cellpose_Plugin_CellProfiler_Instructions/cellprofiler_conda_env.yml

```diff
@@ -0,0 +1,27 @@
+name: cp4
+
+channels:
+- conda-forge
+- anaconda
+- bioconda
+- defaults
+
+dependencies:
+- python=3.8
+- pip
+- numpy
+- matplotlib
+- pandas
+- mysqlclient
+- openjdk
+- scikit-learn
+- mahotas
+- gtk2
+- Jinja2=3.0.1
+- inflect=5.3.0
+- wxpython=4.1.0
+- mysqlclient=1.4.4
+- sentry-sdk=0.18.0
+- pip:
+- cellprofiler==4.2.4
+- cellpose
```

Comment:
Why not use the [conda-forge cellpose](https://anaconda.org/conda-forge/cellpose)?

I also wonder that if you specifically install cellpose 2.0, if the plugin will recognize the 2.0 models (e.g. cyto2)!

Reply:
@gwaybio 

![image](https://user-images.githubusercontent.com/107513215/196213188-23d27b68-aad9-4cb0-8150-6811cc5b5557.png)

I just uninstalled Cellpose to then try to install 2.0 specifically and looks like 2.0 was already installed based on the screenshot. So, I am not sure why CellProfiler is not finding all of the models from 2.0.

Reply:
Conda and pip are two separate package managers that draw from different versions on your machine. It's possible that your cellpose version inside the `cellprofiler` conda environment is version 1

Reply:
@gwaybio 

I think we can conclude that CellProfiler is the issue since I installed cellpose via conda-forge and still isn't working.

I am going to go with pip install because it was much faster at installing than conda-forge when running the .yml files.